src/pybind/mgr/HACKING.md

```diff
@@ -0,0 +1,56 @@
+# Ceph Manager Developer Documentation
+
+## Unit Testing
+
+In the manager our unit tests are based on **tox**.
```

Comment:
Which is actually a bit misleading. We're only testing very few files, not this directory. Instead, this directory mostly consist of files that are tested by individual unit tests not covered here.